site/content/docs/main/config/fundamentals.md

```diff
@@ -151,6 +151,9 @@ Some examples of invalid configurations that Contour provides statuses for:
 - Multiple header conditions of type "exact match" with the same header key.
 - Contradictory header conditions on a route, e.g. a "contains" and "notcontains" condition for the same header and value.
 
+Invalid configuration is ignored and will be not used in the ingress routing configuration.
```

Comment:
I would say enumerating what invalid configurations are "ignored" and 50x responses programmed for would be good

maybe a table that we could expand on in the future, or a couple quick sentences here

Reply:
I agree that we should at least have some examples of invalid config and how this works in those cases. Maybe just describing the scenarios covered by test cases is enough for now.

I don't think that having an exhaustive list is very maintainable long-term, but we should at least give people an idea of what happens and where to look for more info in this bit.